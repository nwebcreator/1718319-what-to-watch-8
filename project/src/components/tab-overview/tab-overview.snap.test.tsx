import { render } from '@testing-library/react';
import TabOverview from './tab-overview';

describe('Component: TabOverview', () => {
  it('should render correctly', () => {
    const film = {
      id: 1,
      name: 'string',
      posterImage: 'string',
      previewImage: 'string',
      backgroundImage: 'string',
      backgroundColor: 'string',
      videoLink: 'string',
      previewVideoLink: 'string',
      description: 'string',
      rating: 10,
      scoresCount: 10,
      director: 'string',
      starring: [],
      runTime: 10,
      genre: 'string',
      released: 2010,
      isFavorite: false,
    };
    const { container } = render(<TabOverview film={film} />);
    expect(container).toMatchSnapshot();
  });
});
