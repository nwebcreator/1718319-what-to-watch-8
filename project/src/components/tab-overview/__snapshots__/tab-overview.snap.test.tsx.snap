// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: TabOverview should render correctly 1`] = `
<div>
  <div
    class="film-rating"
  >
    <div
      class="film-rating__score"
    >
      10
    </div>
    <p
      class="film-rating__meta"
    >
      <span
        class="film-rating__level"
      >
        Awesome
      </span>
      <span
        class="film-rating__count"
      >
        10
         ratings
      </span>
    </p>
  </div>
  <div
    class="film-card__text"
  >
    <p>
      string
    </p>
    <p
      class="film-card__director"
    >
      <strong>
        Director: 
        string
      </strong>
    </p>
    <p
      class="film-card__starring"
    >
      <strong>
        Starring: 
        
      </strong>
    </p>
  </div>
</div>
`;
